### PR TITLE
Spec urls

### PR DESCRIPTION
**Pull request by pypingou**

### CODE REVIEW

File: pkgdb2client/admin.py [hunk 1]
```diff
@@ -391,15 +398,26 @@ def main():
     elif arg.verbose:
         LOG.setLevel(logging.INFO)
 
-    if arg.test:
-        global PKGDBCLIENT
-        print "Testing environment"
+    global PKGDBCLIENT
+    if arg.pkgdburl:
+        print "Querying pkgdb at: %s" % arg.pkgdburl
```
Comment (ralphbean) [inline pkgdb2client/admin.py]: This should probably be a ``log.debug`` statement?

File: pkgdb2client/admin.py [hunk 2]
```diff
@@ -391,15 +398,26 @@ def main():
...
-            insecure=True)
+            arg.pkgdburl,
+            login_callback=pkgdb2client.ask_password)
+
+    PKGDBCLIENT.insecure = arg.insecure
+
+    if arg.bzurl:
+        if not arg.bzurl.endswith('xmlrpc.cgi'):
+            arg.bzurl = '%s/xmlrpc.cgi' % arg.bzurl
+        print "Querying bugzilla at: %s" % arg.pkgdburl
```
Comment (ralphbean) [inline pkgdb2client/admin.py]: Same here.  log.debug instead of print?

File: pkgdb2client/admin.py [hunk 3]
```diff
@@ -391,15 +398,26 @@ def main():
...
+        utils.BZCLIENT.url = arg.bzurl
+        utils.BZCLIENT._sslverify = not arg.insecure
+
+    if arg.fasurl:
+        print "Querying FAS at: %s" % arg.pkgdburl
```
Comment (ralphbean) [inline pkgdb2client/admin.py]: And here.

**Comment by ralphbean** [comment]
Looks good to me, excepting those ``print`` statements.  :+1: pending that.

File: pkgdb2client/admin.py (hunk 1 above)
Comment (pypingou) [inline pkgdb2client/admin.py]: In a way I agree, on the other side this is such an important change (results may/will vary) that I wonder if we shouldn't be explicit about it.

What do you think?

Comment (ralphbean) [inline pkgdb2client/admin.py]: Oh.  Then make it a ``log.info`` statement?

**Comment by ralphbean** [comment]
:+1:  :)

**Comment by pypingou** [comment]
Thanks @ralphbean

**Pull request merged**